Source/Core/VideoCommon/RenderBase.h

```diff
@@ -133,6 +134,8 @@ class Renderer
 	// Max height/width
 	virtual int GetMaxTextureSize() = 0;
 
+	static Common::Event s_screenshotCompleted;
```

Comment:
That's what I was sorta implying; if they both serve the same purpose, the volatile can be tossed out.